src/Core/Components/Label/FluentInputLabel.razor.cs

```diff
@@ -14,6 +14,9 @@ public partial class FluentInputLabel
     /// <summary />
     private IJSObjectReference? Module { get; set; }
 
+    [Parameter]
+    public FluentInputBase<TValue>? Input { get; set; }
```

Comment:
I don't think that is a good idea to link a all input components to a `FluentInputLabel`.
You have `ForId` to do this (of course, only in HTML but what if the `ForId` is a listbox and the `Input` is a checkbox?).

Isn't it possible to do things differently?

Reply:
Look at it more as an addition of a 'shortcut'. If you want the InputLabel to be connected to the input, you can just use `this` for its value and then all the needed other parameters can be gotten from the Input value instead of specifying them all separately. But all the other parameters all still there so nothing is broken/changed from what was there before

Reply:
As agreed, I remove the `Input` parameter an specify the other parameters where needed